usaspending_api/bulk_download/v2/views.py

```diff
@@ -152,6 +153,18 @@ def post(self, request):
             if required_param not in json_request:
                 raise InvalidParameterException('{} parameter not provided'.format(required_param))
 
+        # TODO: Refactor with the bulk_download method in populate_monthly_files.py
+        # Check if the same request has been called today
+        current_date = datetime.date.today()
```

Comment:
Its minor, but it is good practice to explicitly use UTC time on servers. This is documented to be local time

Reply:
Definitely can do.